lib/saml_idp/assertion_builder.rb

```diff
@@ -14,10 +15,12 @@ class AssertionBuilder
     attr_accessor :raw_algorithm
     attr_accessor :authn_context_classref
     attr_accessor :expiry
+    attr_accessor :encryption_opts
+    attr_accessor :encryption_key
```

Comment:
Not strictly, no. Mostly it's for making testing easier, although now I see that I didn't end up using the test where I needed to access the key. I will remove that accessor.